examples/osgi-test-example-bnd/cnf/ext/central.maven

```diff
@@ -0,0 +1,49 @@
+org.osgi:org.osgi.test.junit5:0.10.0
+org.osgi:org.osgi.test.junit5.cm:0.10.0
+org.osgi:org.osgi.test.common:0.10.0
+org.osgi:org.osgi.test.assertj.framework:0.10.0
+org.assertj:assertj-core:3.18.0
```

Comment:
Need to update the versions of assertj, mockito, Junit, bnd, etc. All these versions are somewhat old now.

Reply:
updated all versions